packages/smooth_app/lib/pages/image_crop_page.dart

```diff
@@ -3,86 +3,98 @@ import 'dart:typed_data';
 
 import 'package:crop_your_image/crop_your_image.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 import 'package:image_picker/image_picker.dart';
 import 'package:path_provider/path_provider.dart';
 import 'package:provider/provider.dart';
 import 'package:smooth_app/themes/theme_provider.dart';
+import 'package:smooth_app/widgets/loading_dialog.dart';
 
-class ImageCropPage extends StatelessWidget {
-  ImageCropPage({Key? key}) : super(key: key);
+Future<File?> startImageCropping(BuildContext context) async {
+  final AppLocalizations appLocalizations = AppLocalizations.of(context)!;
+  final Uint8List? bytes = await LoadingDialog.run<Uint8List?>(
+    context: context,
+    future: pickImage(),
+    title: appLocalizations.selecting_photo,
+  );
```

Comment:
I'm not sure I would use a `LoadingDialog` here as the user has control over what's going on: picking an image. Just calling `await picImage();` instead?
But maybe there's something I miss about UX: I never use that part of the app, as I cannot scan.

Reply:
Actually makes sense